en/components/autocomplete.md

```diff
@@ -0,0 +1,200 @@
+
+The following example displays that the positioning of the drop down can be set to be always above the input, where the directive is applied. It also disables opening and closing animations. For that purpose the `ConnectedPositioningStrategy` is used:
+
+```html
+<igx-input-group #inputGroup>
+    <input igxInput name="towns" type="text"
+        [igxAutocomplete]='townsPanel'
+        [igxAutocompleteSettings]='settings'/>
+    <label igxLabel for="towns">Towns</label>
+</igx-input-group>
+<igx-drop-down #townsPanel>
+    <igx-drop-down-item *ngFor="let town of towns">
+        {{town}}
+    </igx-drop-down-item>
+</igx-drop-down>
+```
+
+```typescript
+@Component({
+    selector: 'app-autocomplete-sample',
+    styleUrls: ['autocomplete.sample.css'],
+    templateUrl: `autocomplete.sample.html`
+})
+export class AutocompleteSampleComponent {
+    constructor() {
+        this.towns = [ 'Sofia', 'Plovdiv', 'Varna', 'Burgas'];
+    }
+    @ViewChild('inputGroup', { read: IgxInputGroupComponent }) inputGroup: IgxInputGroupComponent;
+
+    this.settings = {
+        positionStrategy: new ConnectedPositioningStrategy({
+            closeAnimation: null,
+            openAnimation: null,
+            verticalDirection: VerticalAlignment.Top,
+            verticalStartPoint: VerticalAlignment.Top
+        })
+    };
+}
+```
+
+> Note: The default positioning strategy is `AutoPositionStrategy` and drop down is opened according to the available space.
+
+## Application Scenarios
+### Autocomplete with grouped items, placed inside Reactive From
+
+The following sample demonstrates an autocomplete that allows selecting a movie, where movies are grouped by genres. The drop down is bound to hierarchical data and is placed inside a Reactive Form:
+
+<div class="sample-container loading" style="height: 800px;">
+    <iframe id="reactive-form-sample" frameborder="0" seamless width="100%" height="100%" src="{environment:demosBaseUrl}/data-entries/reactive-forms" onload="onSampleIframeContentLoaded(this);"></iframe>
+</div>
+<div>
+    <button data-localize="stackblitz" disabled class="stackblitz-btn" data-iframe-id="reactive-form-sample" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
+</div>
+
+> Note: Follow the [Template Driven Forms Integration](input_group.md) topic to see the same example and an igxAutocomplete placed inside a Template Driven Form.
+<div class="divider--half"></div>
+
+### Autocomplete bound to remote data
+The following sample demonstrates an igxAutocomplete with drop down bound to a remote service. For the purpose of the sample there is a delay, in order to apply a custom template, that will show instead of the drop down, when data is loading:
```

Comment:
Include code snippet showing the declaration of the templates (loading/empty). 
Also Increase the delay to at least 1500ms